ExtractUserDataActivity/handler.ts

```diff
@@ -655,7 +694,8 @@ export function createExtractUserDataActivityHandler({
           messagesView: allUserData.messagesView.map(cleanData),
           notificationStatuses: allUserData.messageStatuses.map(cleanData),
           notifications,
-          profiles: allUserData.profiles.map(cleanData)
+          profiles: allUserData.profiles.map(cleanData),
+          servicesPreferences: allUserData.servicesPreferences
         } as AllUserData;
```

Comment:
not needed cast

Reply:
done in https://github.com/pagopa/io-functions-admin/pull/191/commits/b8fc8ca5c09962851f6e1e5838421523e7643955